projects/batfish/src/test/java/org/batfish/main/BatfishTest.java

```diff
@@ -4,7 +4,6 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.collection.IsEmptyCollection.empty;
-import static org.junit.Assert.assertThat;
```

Comment:
`import static` + just using `assertThat` (rather than `Assert.assertThat`) is generally preferred in unit test code. Just for sanity/cleanliness.